utils/users.js

```diff
@@ -121,24 +121,14 @@ function getPaginationLink(query, cursor, documentId) {
  * @param allUsers {Array} - list of total users from firebase
  *
  */
-function getFilteredUsers(allPRs, allUsers) {
+function getFilteredUsernames(allPRs) {
```

Comment:
`getFilteredUsernames` doesn't tell me much. This is more like a list of users from PRs. What can we name this to make more sense later.

If we were to filter a list from discord messages tomorrow, we won't call that function `getFilteredUsernames2`, right?

Reply:
`getUsernamesFromPRs` can be apt here?

Reply:
We give it a list of PRs, and the function extracts a list of unique usernames.

So, how about `getContributingUsernames`?